### DISCUSSION TITLE
Sponsorship question when sponsoring as organisation

### DISCUSSION THREAD

**Discussion by StarfallProjects** [squidfunk/mkdocs-material#4963] [General]
I'm trying to reduce the bus factor of the docs setup at my company. I'm currently using an access token generated by my account (which is possible because I sponsor individually, as well as my company sponsoring) I spent yesterday trying to sort out a token from the company account - only to realise we're sponsoring as an organisation (n8n-io, which is a GitHub org, and so of course can't generate a token)

Is there a way to access Insiders as an organisation?

**Comment by squidfunk** [comment]
There's yet no canonical way, but we're actively working on providing better access management for organizations! I'd be happy to learn about your workflow. Does your organization work with outside collaborators? Do you have a dedicated docs team on GitHub?

In the meantime we recommend to create a bot account for the organization and fork Insiders to your organization. You can then share access among all members. We understand that this is not ideal, but as said, it's going to get better soon 😊 It was important for us to collect enough use cases in the last two years to build something that solves (almost) all cases when sharing access.

**Reply by StarfallProjects** [reply]
We have forked it, and make this available for internal users to work with locally. However, was unsure how to use this with Netlify (perhaps I just need to get googling, but don't think Netlify can clone a private repo as part of a deploy?). It's not so much sharing access among members (deploy previews substantially reduce the need), as making sure our builds & deploys aren't relying on my personal access.
I'll go look at Netlify's docs, and in the mean time, won't go skydiving. Thanks as always for the swift response.

**Reply by squidfunk** [reply]
You should be able to use a PAT of a bot account that has access to your org/fork for this, if I'm not mistaken.
